src/WPF/ArcGISRuntime.WPF.Viewer/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml

```diff
@@ -14,7 +14,7 @@
                     Source="http://services.arcgisonline.com/ArcGIS/rest/services/World_Topo_Map/MapServer"/>
```

Comment:
The Map is specified twice in this sample — first in XAML then immediately in `Initialize()`. Only one or the other is needed.